drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/panel/AnalysisReportScreenViewImpl.java

```diff
@@ -16,58 +16,58 @@
 
 package org.drools.workbench.screens.guided.dtable.client.widget.analysis.panel;
 
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
-import com.google.gwt.core.client.GWT;
-import com.google.gwt.uibinder.client.UiBinder;
-import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.dom.client.Element;
 import com.google.gwt.user.cellview.client.CellList;
 import com.google.gwt.user.cellview.client.HasKeyboardPagingPolicy;
 import com.google.gwt.user.cellview.client.HasKeyboardSelectionPolicy;
+import com.google.gwt.user.client.DOM;
 import com.google.gwt.user.client.ui.Composite;
 import com.google.gwt.user.client.ui.RequiresResize;
-import com.google.gwt.user.client.ui.ScrollPanel;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.gwt.view.client.ListDataProvider;
 import com.google.gwt.view.client.SelectionChangeEvent;
 import com.google.gwt.view.client.SingleSelectionModel;
+import org.drools.workbench.screens.guided.dtable.client.resources.i18n.AnalysisConstants;
 import org.drools.workbench.screens.guided.dtable.client.widget.analysis.reporting.Issue;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
 
+@Dependent
+@Templated
 public class AnalysisReportScreenViewImpl
         extends Composite
         implements AnalysisReportScreenView,
                    RequiresResize {
 
     private AnalysisReportScreen presenter;
 
-    interface Binder
-            extends
-            UiBinder<Widget, AnalysisReportScreenViewImpl> {
 
-    }
+    @DataField( "progressBar" )
```

Comment:
It was ( I took the escape ), but since you had doubts somebody else might too. It is not the traditional progress bar we are familiar with. I'll change it to avoid confusion.